modules/git/git.go

```diff
@@ -33,16 +33,18 @@ var (
 	// DefaultContext is the default context to run git commands in, must be initialized by git.InitXxx
 	DefaultContext context.Context
 
-	SupportProcReceive bool // >= 2.29
-	SupportHashSha256  bool // >= 2.42, SHA-256 repositories no longer an ‘experimental curiosity’
+	DefaultFeatures struct {
```

Comment:
Maybe just `Features`? The underlaying git will not change so there is only the default?

Reply:
> Maybe just `Features`? The underlaying git will not change so there is only the default?

I think 2 words are easier to search (grep) and maintain. And maybe in the future we need some other non-default features.